package.json

```diff
@@ -54,7 +54,7 @@
     "bignumber.js": "^7.2.0",
     "chalk": "^2.4.1",
     "ethereumjs-abi": "^0.6.5",
-    "openzeppelin-solidity": "~1.9.0",
+    "openzeppelin-solidity": "^1.10.0",
```

Comment:
The dependency was defined as `~` because OpenZeppelin doesn't respect semver in the 1.x release.

We should switch it back to patch updates only, and openzeppelin-zos should be using that as well.